test/DebugInfo/ASTSection-multi.swift

```diff
@@ -0,0 +1,49 @@
+// REQUIRES: OS=linux-gnu
+// REQUIRES: executable_test
+// REQUIRES: swift_tools_extra
+
+// Test that concatenated .swift_ast sections of various sizes can be parsed.
+
+// RUN: %empty-directory(%t)
+
+// RUN: echo "public let a0 = 0"  >%t/a0.swift
+
+// RUN: echo "public let a1 = 0"  >%t/a1.swift
+// RUN: echo "public let b1 = 0" >>%t/a1.swift
+
+// RUN: echo "public let a2 = 0"  >%t/a2.swift
+// RUN: echo "public let b2 = 0" >>%t/a2.swift
+// RUN: echo "public let c2 = 0" >>%t/a2.swift
+
+// RUN: echo "public let a3 = 0"  >%t/a3.swift
+// RUN: echo "public let b3 = 0" >>%t/a3.swift
+// RUN: echo "public let c3 = 0" >>%t/a3.swift
+// RUN: echo "public let d3 = 0" >>%t/a3.swift
+
+// RUN: %target-build-swift %t/a0.swift -c -g -o %t/a0.o -parse-as-library
+// RUN: %target-build-swift %t/a1.swift -c -g -o %t/a1.o -parse-as-library
+// RUN: %target-build-swift %t/a2.swift -c -g -o %t/a2.o -parse-as-library
+// RUN: %target-build-swift %t/a3.swift -c -g -o %t/a3.o -parse-as-library
+
+// RUN: %target-build-swift %t/a0.swift -emit-module -emit-module-path %t/a0.swiftmodule
+// RUN: %target-build-swift %t/a1.swift -emit-module -emit-module-path %t/a1.swiftmodule
+// RUN: %target-build-swift %t/a2.swift -emit-module -emit-module-path %t/a2.swiftmodule
+// RUN: %target-build-swift %t/a3.swift -emit-module -emit-module-path %t/a3.swiftmodule
+
+// RUN: %target-swift-modulewrap %t/a0.swiftmodule -o %t/a0-mod.o
```

Comment:
Does swift-modulewrap build a MachO container for a swiftmodule?

Reply:
Technically that would work, but it's not wired up correctly at the moment and returns an IRGen error "Unable to find a runtime library path".

Reply:
Ah, sorry, my unfamiliarity with this code is showing. What does swift-modulewrap do?

Reply:
it invokes `swift -modulewrap` which is used on Linux (where we don't control the linker) to wrap a `.swfitmodule` inside a `.swift_ast` section in an object file, so it can be linked together with the program. The linker concatenates all the `.swift_ast` sections  for LLDB to be found.